CHANGELOG.md

```diff
@@ -6,4 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/eth-simple-keyring/
+## [5.0.0]
+### Changed
+- Always return rejected Promise upon failure ([#85](https://github.com/MetaMask/eth-simple-keyring/pull/85))
+- **BREAKING:** Makes version-specific `signTypedData` methods private ([#84](https://github.com/MetaMask/eth-simple-keyring/pull/84))
```

Comment:
Should we list these under "Removed"? :thinking: I guess either way is fine.

Reply:
I could see it either way!